src/components/hooks/useParticipants.js

```diff
@@ -20,7 +20,7 @@ export default function useParticipants(membershipID) {
       console.error(error.message);
     };
     const onMembers = (data) => {
-      setParticipants(data.members);
+      setParticipants(data.members.slice());
```

Comment:
Can you use the spread operator instead?
```suggestion
      setParticipants([...data.members]);
```

Reply:
Note that either approach makes a shallow copy of `members`, but i think it should be ok

Reply:
cool